client/src/tests/data-formats/data-formats.test.ts

```diff
@@ -1,13 +1,12 @@
 import { IDataSourceDictionary } from '../../data-sources';
-import { setupTests } from '../utils/setup';
 import Sample from '../../data-sources/plugins/Sample';
 
 import { formatTests } from './formats';
 import * as formats from '../../utils/data-formats';
 
 describe('Data Formats', () => {
 
```

Comment:
consider remove this extra line

Reply:
I think in this case it would be easier to read it this way